src/main/java/nl/tudelft/pixelperfect/event/EventParameter.java

```diff
@@ -0,0 +1,72 @@
+package nl.tudelft.pixelperfect.event;
+
+/**
+ * Class representing a single parameter of an Event, having either a discrete value
+ * (EventParameterValue) or a continuous one (Integer).
+ * 
+ * @author Jesse Tilro
+ *
+ */
+public class EventParameter {
+
+  private String key;
+  private EventParameterValues value;
+  private int number;
```

Comment:
Since `value` and `number` are practically alternatives and not complementary, I believe their names should be fine like this, right?